mozilla_version/maven.py

```diff
@@ -54,3 +54,21 @@ def _compare(self, other):
             return -1
         else:
             return 0
+
+    @property
+    def is_beta(self):
+        """Maven versions can never be Beta."""
+        return False
+
+    @property
+    def is_release_candidate(self):
+        """Maven version can never be Release Candidates."""
+        return False
+
+    @property
+    def is_release(self):
+        """Return `True` if the others are both False."""
+        return not any((
+            self.is_beta, self.is_release_candidate,
```

Comment:
Nit: Add `self.is_snapshot` to this list. They're part of the Maven version scheme.

```suggestion
            self.is_beta, self.is_release_candidate, self.is_snapshot
```